images/manageiq-base/Dockerfile

```diff
@@ -36,6 +37,9 @@ LABEL name="manageiq-base" \
       io.k8s.description="ManageIQ is a management and automation platform for virtual, private, and hybrid cloud infrastructures." \
       io.openshift.tags="ManageIQ,miq,manageiq"
 
+COPY rpms/* /tmp/rpms/
```

Comment:
I'm ok with merging as is for now, but can we prefer a volume mount instead?  That way, we don't get the RPMs in one image layer only to remove them in a later image layer.

Reply:
Can we do this all the RUN command then? I think that would solve the problem, right?

Reply:
Oh, we need them in the other images .... Nevermind.

Reply:
I definitely want to get rid of adding RPMs in base and removing in ui. If you can think of a better way than duplicating that part of the codes, let me know!

Reply:
I think volume mount isn't supported during build?